winpr/include/winpr/pool.h

```diff
@@ -110,7 +110,7 @@ typedef VOID (*PTP_WIN32_IO_CALLBACK)(PTP_CALLBACK_INSTANCE Instance, PVOID Cont
 
 #endif
 
-#if (!defined(_WIN32) || ((defined(_WIN32) && (_WIN32_WINNT < 0x0600))))
+#if (!defined(_WIN32) || ((defined(_WIN32) && (_WIN32_WINNT < 0x0600))) || ((defined(_WIN32) && defined(__MINGW32__))))
```

Comment:
Unnecessary

Reply:
Mingw64 system headers/libraries doesn't have ThreadPool* API functions as of now. This is the only way to make it work(Using implementation of older APIs) until those functions will be added(Which I doubt of - they exist since Vista and still not there...).